ServerCore/Pages/Puzzles/Play.cshtml.cs

```diff
@@ -94,6 +97,12 @@ public async Task OnGetAsync(
                 {
                     AnyErrata = VisibleTeamPuzzleViews.Any(v => v.Errata != null);
                 }
+
+                if (Event.HasPlayerClasses)
+                {
+                    TeamMembers tm = _context.TeamMembers.Where(tm => tm.Team == Team && tm.Member == LoggedInUser).FirstOrDefault();
+                    LoggedInPlayerClass = PlayerClassHelper.GetActiveClassForPlayer(tm)?.UniqueName ?? "";

```

Comment:
Can this handle null if the player's class isn't set?

Reply:
It does now